psp-examples/private-compressed-account/programs/private-compressed-account/src/psp_accounts.rs

```diff
@@ -0,0 +1,143 @@
+use crate::processor::TransactionsConfig;
+use anchor_lang::prelude::*;
+use anchor_spl::token::Token;
+use light_verifier_sdk::{light_transaction::VERIFIER_STATE_SEED, state::VerifierState10Ins};
+use merkle_tree_program::transaction_merkle_tree::state::TransactionMerkleTree;
+use merkle_tree_program::utils::constants::TOKEN_AUTHORITY_SEED;
+use merkle_tree_program::{program::MerkleTreeProgram, EventMerkleTree};
+use verifier_program_two::{self, program::VerifierProgramTwo};
+
+// Send and stores data.
+#[derive(Accounts)]
+pub struct LightInstructionFirst<'info, const NR_CHECKED_INPUTS: usize> {
+    /// First transaction, therefore the signing address is not checked but saved to be checked in future instructions.
+    #[account(mut)]
+    pub signing_address: Signer<'info>,
+    pub system_program: Program<'info, System>,
+    #[account(init, seeds = [&signing_address.key().to_bytes(), VERIFIER_STATE_SEED], bump, space= 3000, payer = signing_address )]
+    pub verifier_state: Account<'info, VerifierState10Ins<NR_CHECKED_INPUTS, TransactionsConfig>>,
+}
+
+#[derive(Debug)]
+#[account]
+pub struct InstructionDataLightInstructionFirst {
+    pub public_amount_spl: [u8; 32],
+    pub input_nullifier: [[u8; 32]; 4],
+    pub output_commitment: [[u8; 32]; 4],
+    pub public_amount_sol: [u8; 32],
+    pub transaction_hash: [u8; 32],
+    pub root_index: u64,
+    pub relayer_fee: u64,
+    pub encrypted_utxos: Vec<u8>,
+}
+
+#[derive(Accounts)]
+pub struct LightInstructionSecond<'info, const NR_CHECKED_INPUTS: usize> {
+    /// First transaction, therefore the signing address is not checked but saved to be checked in future instructions.
+    #[account(mut)]
+    pub signing_address: Signer<'info>,
+    #[account(mut, seeds = [&signing_address.key().to_bytes(), VERIFIER_STATE_SEED], bump)]
+    pub verifier_state: Account<'info, VerifierState10Ins<NR_CHECKED_INPUTS, TransactionsConfig>>,
+}
+
+/// Executes light transaction with state created in the first instruction.
+#[derive(Accounts)]
+pub struct LightInstructionThird<'info, const NR_CHECKED_INPUTS: usize> {
```

Comment:
Heh, the `light_accounts` macro is still not applied here, looking forward to do this in other PR :D